public/dist/data/benefits.json

```diff
@@ -0,0 +1,25 @@
+{
+  "benefits": [
+    {
+      "headline": "Digital and Online Marketing",
+      "description":
+		"Bacon ipsum dolor amet jowl sirloin swine biltong shoulder shank hamburger alcatra. Cupim shank prosciutto, sausage spare ribs shoulder ball tip kielbasa beef ribs. Brisket biltong fatback short ribs. Drumstick t-bone rump capicola ham, burgdoggen tail pork loin.",
```

Comment:
Mmm... prosciutto.

(I assume this is temporary?)

Reply:
yes this is a prototype